Zend/zend_inheritance.c

```diff
@@ -1154,8 +1154,8 @@ static void zend_add_trait_method(zend_class_entry *ce, const char *name, zend_s
 			/* Make sure the abstract declaration is compatible with previous declaration */
```

Comment:
But no, that doesn't really make sense to me. Allowing that code should be fine.